src/skeletons.tsx

```diff
@@ -0,0 +1,184 @@
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase />
+            <SkeletonBase />
+            <SkeletonBase width="75%" />
+        </Stack>
+    );
+};
+
+type SkeletonCircleProps = Omit<SkeletonLineProps, 'width'> & {size?: number | string};
+
+const SkeletonCircle = ({
+    disableAnimation = false,
+    ariaLabel,
+    size = 40,
+}: SkeletonCircleProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({width: size, height: size});
+
+    return (
+        <div
+            className={classNames(animationClasses.animation, styleClass.wrap)}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase height={size} width={size} radius="50%" />
+        </div>
+    );
+};
+
+const useSkeletonRowStyles = createUseStyles(() => ({
+    row: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+    circle: {
+        flexShrink: 0,
+        flexGrow: 0,
+    },
+    line: {
+        flexShrink: 1,
+        marginLeft: 16,
+    },
+}));
+
+type SkeletonRowProps = SkeletonLineProps;
+
+const SkeletonRow = ({
+    width = '100%',
+    disableAnimation = false,
+    ariaLabel,
+}: SkeletonRowProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({width, height: 'content-initial'});
+    const rowClass = useSkeletonRowStyles();
+
+    return (
+        <div
+            className={classNames(animationClasses.animation, styleClass.wrap, rowClass.row)}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase height={40} width={40} radius="50%" className={rowClass.circle} />
+            <SkeletonBase width={width} className={rowClass.line} />
+        </div>
+    );
+};
+
+type SkeletonRectangleProps = SkeletonLineProps & {height: number | string};
+
+const SkeletonRectangle = ({
+    width = '100%',
+    height = '100%',
+    disableAnimation = false,
+    ariaLabel,
+}: SkeletonRectangleProps): JSX.Element => {
+    const animationClasses = useAnimation({disableAnimation});
+    const styleClass = useSkeletonStyles({width, height: 'content-initial'});
+
+    return (
+        <div
+            className={classNames(animationClasses.animation, styleClass.wrap)}
+            role="status"
+            aria-hidden={ariaLabel === undefined}
+            aria-busy={ariaLabel !== undefined}
+            aria-label={ariaLabel}
+        >
+            <SkeletonBase height={height} width={width} />
+        </div>
+    );
+};
+export {SkeletonLine, SkeletonText, SkeletonCircle, SkeletonRow, SkeletonRectangle};
```

Comment:
any reason to not declare the `export` in the same line where the components are declared?

Reply:
no, simply coding routine. I can change it